samples/connectedhomeip/light_bulb/src/light_bulb_publish_service.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include "light_bulb_publish_service.h"
+
+#include "app_task.h"
+
+#include <logging/log.h>
+#include <zephyr.h>
+
+#include <inet/InetConfig.h>
+#include <platform/CHIPDeviceLayer.h>
+#include <transport/TransportMgr.h>
+#include <transport/raw/UDP.h>
+
+LOG_MODULE_DECLARE(app);
+
+const char kPublishServicePayload[] = "chip-light-bulb";
+const char kPublishDestAddr[] = "ff02::1";
```

Comment:
Thi should be changed to `ff03::1`. After that, we should also update the doc, changing `fe80..` to `fd..`.